src/Blammo/Logging/Logger.hs

```diff
@@ -49,7 +51,33 @@ import System.Log.FastLogger.Compat
   , newStderrLoggerSetN
   , newStdoutLoggerSetN
   )
-import UnliftIO.Exception (throwString)
+import UnliftIO (MonadUnliftIO)
+import UnliftIO.Exception (finally, throwString)
+
+-- | Create a new 'Logger', pass it to a callback, and ensure
+--   that the logger is flushed afterward
```

Comment:
I might be more intention-revealing here. In case were ever do more than flush (such as close file-handlers). Then it may make sense to add that not about usage pitfalls.

```suggestion
-- | Create a new 'Logger', pass it to a callback, and clean it up
--
-- Applications should avoid calling this more than once in their lifecycle.
```

Reply:
Added the same comment to `runLoggerLoggingT` as well.